src/test/java/org/zipcodewilmington/lab1/ArrayUtilities.java

```diff
@@ -0,0 +1,73 @@
+package org.zipcodewilmington.lab1;
+
+public class ArrayUtilities {
+
+
+    public static void moveLastToFirst(int[] values) {
+
+        if (values.length < 2) return;
+
+        int last = values[values.length - 1];
+
+        for(int i = values.length - 1; i > 0; i--) {
+            values[i] = values[i - 1];
+        }
+        values[0] = last;
+    }
+
+    public static int[] removeDuplicate(int[] values) {
+        int j = 0;
```

Comment:
I would move this below the if statement because it is not being use before the if statement. You should also named these variable to something more descriptive.